data/tokens.json

```diff
@@ -3407,6 +3351,63 @@
         "textCase": "{gtmds_text-case-none}"
       },
       "type": "typography"
+    },
+    "gtmds_popover_margin": {
+      "value": "{gtmds_space-xs}",
```

Comment:
```suggestion
      "value": "{gtmds_size-lg}",
```

Reply:
With lg the space looks really very big, I'll be happy to talk why this change is needed?

Reply:
size-lg , not space :) we are using the size tokens only the space ones were intended for composites but we never got to that point so we use only size across the board